scripts/release/index.js

```diff
@@ -38,6 +38,7 @@ cooker.cook('publish', [
       cook.pushTagToRemote,
       cook.createReleaseNotes(releaseNotesTemplate),
       cook.createGithubRelease,
+      cook.runNpmScript('deploy'),
```

Comment:
I have no idea how it works. You did the script and @gaspard did the new setup stuff :) @gaspard , you have any comments here?